internal/collectors/virtualization.go

```diff
@@ -22,14 +22,10 @@ func (Virtualization) run(arch Architecture) (Result, error) {
 
 	output, err := util.Execute([]string{systemdDetectVirtExecutable, "-v"}, []int{0, 1})
 
-	if err != nil {
+	if err != nil || string(output) == "none" {
```

Comment:
Just realized about this `"none"` string, which is pretty wild. Could you document where this is coming from?

Reply:
I added a comment addressing where the "none" stems from.